### DISCUSSION TITLE
Elebest Navigationssysteme / Elebest navigation systems

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#598] [Q&A] [answered]
Hallo zusammen,

hat jemand hier ein Elebest mit Cruiser im Einsatz und Erfahrungswerte? 

Hello everyone,

does anyone here have an Elebest with cruiser in use and experience?

**Comment by airald** [comment]
Hello, 

this post will help you.
https://github.com/devemux86/cruiser/discussions/508

**Reply by ToToGSARider** [reply, marked as answer]
Elebest bietet auch ein Gerät an, das ein Android Betriebssystem hat. Du kannst auf dem Elebest direkt installieren. 
Ich dachte, dass es hier schon Anwender gibt, die so ein Gerät haben. 

Elebest also offers a device that has an Android operating system. You can install directly on the Elebest. 
I thought that there are already users here who have such a device.

**Reply by airald** [reply]
Very interesting! I only saw the android auto feature.

**Reply by Leeb64** [reply]
it only has Android 6!
Not so current anymore.

**Comment by devemux86** [comment]
> it only has Android 6!

Note that old Androids up to 7 (Nougat) have trouble communicating with https websites.
(browsers can usually work because they use their own engines)

Because internet is updating the https protocol but old Androids do not receive the update.
Cruiser can get around this, but this is an indication of Android fragmentation and support.
